core/templates/components/state-editor/state-responses-editor/state-responses.component.html

```diff
@@ -159,7 +159,7 @@ <h3 class="oppia-exp-answer-card-header" tabindex="0">Learner's Answers and Oppi
         <div *ngIf="!isInQuestionMode()" class="solicit-answer-checkbox">
           <span *ngIf="!isCurrentInteractionTrivial()" [hidden]="!enableSolicitAnswerDetailsFeature" class="solicit-answer-details-span">
             <input id="label-target-solicit-answer-details-checkbox" type="checkbox" class="e2e-test-solicit-answer-details-checkbox" (change)="onChangeSolicitAnswerDetails()" [(ngModel)]="stateSolicitAnswerDetailsService.displayed" [disabled]="!editabilityService.isEditable()">
-            <label for="label-target-solicit-answer-details-checkbox">Solicit Answer Details</label>
+            <label for="label-target-solicit-answer-details-checkbox">&nbsp;Solicit Answer Details</label>
```

Comment:
Could we avoid &nbsp; here and elsewhere? Use CSS for styling, instead.

Reply:
@seanlip Yeah, will do it via css.